Lib/idlelib/idle_test/test_sidebar.py

```diff
@@ -695,7 +695,8 @@ def test_mousewheel(self):
         delta = -1 if sys.platform == 'darwin' else 120
         sidebar.canvas.event_generate('<MouseWheel>', x=0, y=0, delta=delta)
         yield
-        self.assertIsNone(text.dlineinfo(text.index(f'{last_lineno}.0')))
+        if sys.platform != 'darwin':  # .update_idletasks() does not work.
```

Comment:
Ordinarily I would ask if this test should continue to run on XQuartz if the problem is only on Aqua:

```suggestion
        if text._windowingsystem != 'aqua':  # .update_idletasks() does not work.
```

But this test was already failing on Aqua and XQuartz due to incorrectly negative `delta`. So this failure may have nothing to do with `.update_idletasks()`, and the test should not be skipped on Aqua or XQuartz. See #103820, #103821